### ISSUE TITLE
Bug detected

### ISSUE THREAD

**Issue by Champion-GuanWang** [SysBioChalmers/GECKO#305]
#### Reproducing these results:

```matlab
% If you worked on matlab, paste your code here
```

#### System information

* Operating system: 
* MATLAB version: 
* GECKO version: 

**I hereby confirm that I have:**
- [ ] Tested my code with [all requirements](https://github.com/SysBioChalmers/GECKO) for running GECKO
- [ ] Done this analysis in the `main` branch of the repository
- [ ] Checked that a similar issue does not exist already

**Comment by mihai-sysbio** [comment]
@Champion-GuanWang thanks for trying out the latest GECKO, for the bug report and for including the script. Could I ask you to provide a few more details on the exact versions of RAVEN, Matlab, and solver?

**Comment by mihai-sysbio** [comment]
Also, do I understand correctly that you had no issues with `runDLKcat` in Docker?

**Comment by Champion-GuanWang** [comment]
> Also, do I understand correctly that you had no issues with `runDLKcat` in Docker?
Yes, we had no issues with  'runDLKcat' in Docker. More details about Raven, Matlab and solver are as follows:
Matlab R2022a academic use; Raven 2.8.0; Set RAVEN solver gurobi 9.5.2. Thank you for debugging!

**Comment by Champion-GuanWang** [comment]
> @Champion-GuanWang thanks for trying out the latest GECKO, for the bug report and for including the script. Could I ask you to provide a few more details on the exact versions of RAVEN, Matlab, and solver?
I am wondering there are any updates about the debugging? We are eager to have your reply. Thank you!

**Comment by edkerk** [comment]
Your `solEC.x` is probably empty? The FBA that was run to generate `solEC` did not find a valid solution.

**Comment by mihai-sysbio** [comment]
I was able to reproduce the issue on macOS, Matlab 2021b, RAVEN 2.8.0 and Gurobi 10, by running `protocol.m` in [this commit](ad2a5930dff80d3eefbbc3e101b5545f6739dc2f):
```
STEP 23 Compare fluxes from ecModel and starting model

solEC = 

  struct with fields:

       x: []
       f: []
    stat: -1
     msg: 'The problem is infeasible'
```

**Comment by edkerk** [comment]
So the problem is the tutorial, not the function. 

We will be editing the tutorial during the next 1-2 weeks, to address this and some other aspects related to resubmission of the GECKO3 manuscript. This issue will be resolved as part of this process.

**Comment by edkerk** [comment]
The protocol was tested again in #310 and is now functional. Some changes might still occur before the next release, but this Issue is resolved.

**Comment by Mengzhen-Li-sw** [comment]
Hi, Running DLKcat is taking an extremely long time, almost 20 hours for a single sample. Could this be related to my Mac platform? Are there any solutions to improve the performance?

**Comment by edkerk** [comment]
@Mengzhen-Li-sw Can you please start a [new Issue](https://github.com/SysBioChalmers/GECKO/issues/new/choose), so that we can keep track of separate concerns / problems / questions? Thanks!
